### PR TITLE
Base

### PR DESCRIPTION
**Pull request by enriquesouza**
Base uses very low GWEI so we need to use it as WEI all the time
https://github.com/voltix-vault/voltix/issues/155

Refactoring EVM so it can support all L1 and L2.

### CODE REVIEW

**Comment by enriquesouza** [comment]
I've added all layer 2

On my tests:

Arbitrum:
Send ETH OK
Send ARB - OK

Optimism:
Send ETH OK
Sedn OP OK

Base:
Send ETH OK
Send USDC OK

Matic:
Send MATIC OK
Send WETH Fixing 

BSC:
SEND BNB OK

**Comment by enriquesouza** [comment]
0x0eA1aeFcB85ae573f0B7CD20AD6D980Cf55ea4d8

BASE tests:
https://basescan.org/address/0x0eA1aeFcB85ae573f0B7CD20AD6D980Cf55ea4d8

BNB tests:
https://bscscan.com/address/0x0eA1aeFcB85ae573f0B7CD20AD6D980Cf55ea4d8

Arbitrum tests:
https://arbiscan.io/address/0x0eA1aeFcB85ae573f0B7CD20AD6D980Cf55ea4d8

Optimism tests:
https://optimistic.etherscan.io/address/0x0eA1aeFcB85ae573f0B7CD20AD6D980Cf55ea4d8

Polygon tests:
https://polygonscan.com/address/0x0eA1aeFcB85ae573f0B7CD20AD6D980Cf55ea4d8

File: voltixApp/VoltixApp/View Models/SendTransaction.swift [hunk 1]
```diff
@@ -214,23 +154,19 @@ class SendTransaction: ObservableObject, Hashable {
...
             "amountInLamports: \(amountInLamports)",
             "amountInSats: \(amountInSats)",
             "feeInSats: \(feeInSats)",
             "amountDecimal: \(amountDecimal)",
             "amountInCoinDecimal: \(amountInCoinDecimal)",
             "gasDecimal: \(gasDecimal)",
-            "gasFeePredictionForEvm: \(gasFeePredictionForEvm)",
-            "gasFeePredictionForEvmFiat: \(gasFeePredictionForEvmFiat)"
+//            "gasFeePredictionForEvm: \(gasFeePredictionForEvm)",
+//            "gasFeePredictionForEvmFiat: \(gasFeePredictionForEvmFiat)"
```
Comment (johnnyluo) [inline voltixApp/VoltixApp/View Models/SendTransaction.swift:169]: can you please remove it?

**Review by johnnyluo — APPROVED** [review]
LGTM

File: voltixApp/VoltixApp/View Models/SendTransaction.swift (hunk 1 above)
Comment (enriquesouza) [reply voltixApp/VoltixApp/View Models/SendTransaction.swift:169]: Sure.

Comment (enriquesouza) [reply voltixApp/VoltixApp/View Models/SendTransaction.swift]: Removed

**Pull request merged**